ghscraper.py

```diff
@@ -253,21 +253,26 @@ def scrapePullRequestComments(repo, repoPath):
                       datetime.datetime.now() + datetime.timedelta(minutes = GITHUB_RATELIMIT_INTERVAL))
                 time.sleep(60*GITHUB_RATELIMIT_INTERVAL)
 
+def __login(args):
```

Comment:
Hi, @sarahsharp! I defer to you on this; I haven't touched Python since 2009. :)